sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -0,0 +1,547 @@
+   * @param {string}             subscription                                        A string object that represents the name of the subscription to retrieve.
+   * @return {Promise<HttpOperationResponse}
+   */
+  getSubscription(topicPath: string, subscriptionPath: string): Promise<HttpOperationResponse> {
+    const fullPath = ServiceBusAtomManagementClient.getSubscriptionPath(
+      topicPath,
+      subscriptionPath
+    );
+    return this._getResource(fullPath, this.subscriptionResourceSerializer);
+  }
+
+  /**
+   * Returns a list of subscriptions.
+   *
+   * @param {string}             topicPath                                 A string object that represents the name of the topic for the subscriptions to retrieve.
+   * @param {object}             [options]                                 The request options.
+   * @param {int}                [options.top]                             The number of topics to fetch.
+   * @param {int}                [options.skip]                            The number of topics to skip.
+   * @return {Promise<HttpOperationResponse}
+   */
+  async listSubscriptions(topicPath: string, options?: any): Promise<HttpOperationResponse> {
+    return this._listResources(
+      topicPath + "/Subscriptions/",
+      options,
+      this.subscriptionResourceSerializer
+    );
+  }
+
+  /**
+   * Creates a rule.
+   *
+   * @param {string}             topicPath                                       A string object that represents the name of the topic for the subscription.
+   * @param {string}             subscriptionPath                                A string object that represents the name of the subscription for which the rule will be created.
+   * @param {string}             rulePath                                        A string object that represents the name of the rule to be created.
+   * @param {object}             [options]                                       The request options.
+   * @param {string}             [options.trueFilter]                            Defines the expression that the rule evaluates as a true filter.
+   * @param {string}             [options.falseFilter]                           Defines the expression that the rule evaluates as a false filter.
+   * @param {string}             [options.sqlExpressionFilter]                   Defines the expression that the rule evaluates. The expression string is interpreted as a SQL92 expression which must evaluate to True or False. Only one between a correlation and a sql expression can be defined.
+   * @param {string}             [options.correlationIdFilter]                   Defines the expression that the rule evaluates. Only the messages whose CorrelationId match the CorrelationId set in the filter expression are allowed. Only one between a correlation and a sql expression can be defined.
```

Comment:
There is no such thing as a correlation "id" that the user can provide. There is such a thing as "correlation filter" that the user can provide, but it most probably will not be a string. This needs more investigation.

Reply:
Based on docs at https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicebus.messaging.correlationfilter?view=azure-dotnet https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicebus.messaging.ruledescription?view=azure-dotnet

Looks like the parameter would be a 'correlationFilter' expression is a `string` since the constructor expects one, and will likely expect a correlation Id as well